ITEM_en.tsv

```diff
@@ -4676,7 +4676,7 @@ ITEM_20150428_005062	The first revelation found in the Crystal Mine. Readable.
 ITEM_20150428_005063	Pondel Soul
 ITEM_20150428_005064	The second revelation found in the Tenet Cathedral. Readable.
 ITEM_20150428_005065	Lump of Kepo Meat
-ITEM_20150428_005066	A lump of meat obtained from Kepo. No sane human would ever eat it.
+ITEM_20150428_005066	Meat Loaf that can be easily obtained from Kepo. However, you can't eat it as is.
```

Comment:
It is not a meat loaf, it is a lump of meat. Also the second sentence is wrong, the Korean explicitly states that if you consider yourself a human being you would never eat it. My translation was a paraphrase of that.